lib/encrypt_env_variables.js

```diff
@@ -0,0 +1,37 @@
+import request from 'superagent-promise';
+import openpgp from 'openpgp';
+import Debug from 'debug';
+
+let debug = Debug('docker-worker-ci:encryptEnvVariable');
+
+export async function encryptEnvVariables(task, envs, publicKeyUrl) {
```

Comment:
I think you'd be more than welcome to propose a variant of this we can fit into "taskcluster-client" :)